<launch>
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
        <arg name="port" value="19871" />
    </include>
    <node name="map_nav_srv" pkg="dji_ros" type="map_nav_srv" output="screen"/>
</launch>
